src/scss/components/_toast.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Toast Component
+ * =================================
+ * Used for ...
+ *
+ * *The `c-toast` component is an optional mixin within Fozzie — if you'd like to use it in your project you can include it by adding `@include toast();` into your SCSS dependencies file.*
+ *
+ * Documentation:
+ * https://fozzie.just-eat.com/styleguide/ui-components/toast
+ */
+
+$toast-radius           : 8px !default;
+$toast-textColor        : $white;
+$toast-bgColor--default : $grey--darkest;
+$toast-bgColor--alert   : $red--dark;
+
+@mixin toast() {
+
+        @keyframes toastSlideUp {
+            0% {
+                top: 0;
+            }
+            100% {
+                top: -90px;
```

Comment:
The heights are currently fixed so that the CSS animation transitions can work. I'm open to suggestions on ways to do this better.

Reply:
would a `translateY` work better than fixed pixels here? 🤔

Reply:
Would be good to pull this out into a variable too, as would save having to update the value in multiple places if it does change.

You could link it to the height of the `c-toast` element then too if you know it always needs to pop up `5px` more than the height you could use `$toast-height: 85px` and `+ 5` where `90px` is used atm

Reply:
@sonnyprince `translateY` does work better as it has removed the jank from the animation, nice one!

@ashleynolan That def make sense, will update now.

Reply:
For future ref: https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/